style/properties/longhands/position.mako.rs

```diff
@@ -133,45 +118,45 @@ ${helpers.single_keyword(
     )}
 % endif
 
-% if engine == "servo":
-    // FIXME: Update Servo to support the same Syntax as Gecko.
-    ${helpers.single_keyword(
-        "align-content",
-        "stretch start flex-start end flex-end center space-between space-around space-evenly",
-        engines="servo",
-        servo_pref="layout.flexbox.enabled",
-        extra_prefixes="webkit",
-        spec="https://drafts.csswg.org/css-align/#propdef-align-content",
-        animation_value_type="discrete",
-        servo_restyle_damage="reflow",
-        affects="layout",
-    )}
+${helpers.predefined_type(
+    "align-content",
+    "AlignContent",
+    "specified::AlignContent(specified::ContentDistribution::normal())",
+    engines="gecko servo",
+    spec="https://drafts.csswg.org/css-align/#propdef-align-content",
+    extra_prefixes="webkit",
+    animation_value_type="discrete",
+    servo_restyle_damage="reflow",
+    affects="layout",
+)}
 
-    ${helpers.single_keyword(
-        "align-items",
-        "stretch flex-start flex-end center baseline",
-        engines="servo",
-        servo_pref="layout.flexbox.enabled",
-        extra_prefixes="webkit",
-        spec="https://drafts.csswg.org/css-flexbox/#align-items-property",
-        animation_value_type="discrete",
-        servo_restyle_damage="reflow",
-        affects="layout",
-    )}
-% endif
-% if engine == "gecko":
-    ${helpers.predefined_type(
-        "align-content",
-        "AlignContent",
-        "specified::AlignContent(specified::ContentDistribution::normal())",
-        engines="gecko",
-        spec="https://drafts.csswg.org/css-align/#propdef-align-content",
-        extra_prefixes="webkit",
-        animation_value_type="discrete",
-        servo_restyle_damage="reflow",
-        affects="layout",
-    )}
+${helpers.predefined_type(
+    "align-items",
+    "AlignItems",
+    "specified::AlignItems::normal()",
+    engines="gecko servo",
+    spec="https://drafts.csswg.org/css-align/#propdef-align-items",
+    extra_prefixes="webkit",
+    animation_value_type="discrete",
+    servo_restyle_damage="reflow",
+    affects="layout",
+)}
+impl_align_conversions!(crate::values::specified::align::AlignItems);
```

Comment:
Why is this added?

Reply:
It was already present in the Gecko version, I just made it unconditional (this doesn't show up very well in the diff due to unnesting). It adds conversions to and from u8. I'm not entirely sure why that's useful, but it didn't seem particularly gecko-specific. I could make it conditional again if that's preferred.